examples/example-vite-svelte-recs/src/dojo/typescript/models.gen.ts

```
@@ -0,0 +1,193 @@
+export interface FieldLayout {
+  selector: BigInt;
+  layout: Layout;
+}
+export const FieldLayoutDefinition = {
+  selector: RecsType.BigInt,
+  layout: LayoutDefinition,
+};
+
+// Type definition for `dojo_starter::models::Moves` struct
+export interface Moves {
+  player: BigInt;
```

Biome

[error] Don't use 'BigInt' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'bigint' instead


(lint/complexity/noBannedTypes)